### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by karam084**
-* Project requirements

- You should implement the following interactions:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- You should implement popups for both mobile and desktop screen sizes:
- Mobile popup
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to the source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -270,6 +270,43 @@ <h4 class="more_det1 more-de2">LET’S CONNECT</h4>
...
+              See Live<img src="./media-icons/Icon.png" alt="media icon"class="media-pop">
+            </button>
+            <button type="button" id="see-source-btn" onclick="">
+              See Source<img src="./media-icons/Vector-pop.png" alt="media icon"class="media-pop">
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    </div>
```
Comment (mmsesay) [inline index.html:308]: - [x] Your pop up window is not well aligned as guided in the Figma design. The popup window should be at the centre. Yours is currently on the left side of the page. See screenshot below:

Your pop up window desktop |
:--------------------------:|
c1

Your pop up window mobile |
:--------------------------:|
c3

File: index.html [hunk 2]
```diff
@@ -270,6 +270,43 @@ <h4 class="more_det1 more-de2">LET’S CONNECT</h4>
...
+          <div class="popup-project-buttons">
+            <button type="button" id="see-live-btn" onclick="">
+              See Live<img src="./media-icons/Icon.png" alt="media icon"class="media-pop">
+            </button>
+            <button type="button" id="see-source-btn" onclick="">
+              See Source<img src="./media-icons/Vector-pop.png" alt="media icon"class="media-pop">
+            </button>
+          </div>
+        </div>
+      </div>
```
Comment (mmsesay) [inline index.html:306]: - [x] Same on your pop up window, your contents are too thin to read. Can you please increase the font-size.
- [x] In addition, your buttons should link to your project live version and your project's GitHub source code

screenshot |
:--------------:|
c2

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @karam084  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @karam084 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**